.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,49 @@
+name: "Bug report"
+description: "Create a report on bugs you find"
+labels:
+  - "bug"
+
+body:
+  - type: checkboxes
+    id: confirmation
+    attributes:
+      label: Before reporting
+      description: Before reporting, you have to check follows
+      options:
+        - label: Checked [same issue](https://github.com/approvers/pulsate/issues?q=) already has not already been reported
```

[must] いらないクエリがついている気がします
```suggestion
        - label: Checked [same issue](https://github.com/approvers/pulsate/issues) already has not already been reported
```

---

このクエリは意図的な設定です. (すでに修正されている PR/Issue を含めて検索するにはクエリを設定しないと普通に `is:open` で検索されてしまう)

参考: https://github.com/approvers/OreOreBot2/pull/672#discussion_r1071445094